RUFAS/biophysical/manure/digester/anaerobic_digester.py

```diff
@@ -100,58 +75,42 @@ def process_manure(self, conditions: CurrentDayConditions, time: RufasTime) -> d
         """Digests manure received on the current day."""
         if self._manure_in_digester.is_empty is True:
             self._report_anaerobic_digester_outputs(
-                biogas=0.0,
-                biogas_energy_content=0.0,
-                evaporated_water=0.0,
-                heating_input_energy=0.0,
-                methane_generation_volume=0.0,
-                methane_leakage_mass=0.0,
-                minimum_digester_volume=0.0,
-                top_cover_volume=0.0,
+                captured_biogas_volume=0.0,
+                captured_methane_volume=0.0,
+                methane_leakage_volume=0.0,
                 simulation_day=time.simulation_day,
             )
             return {}
 
-        moisture_fraction = self._manure_in_digester.water / self._manure_in_digester.mass
-        specific_input_energy = self._calculate_specific_input_energy(
-            conditions.mean_air_temperature, moisture_fraction, self._temperature_set_point
-        )
-        heating_input_energy = (
-            specific_input_energy * self._manure_in_digester.volume * GeneralConstants.LITERS_TO_CUBIC_METERS
-        )
-
-        minimum_digester_volume = self._manure_in_digester.volume * self._hydraulic_retention_time
-        top_cover_volume = minimum_digester_volume * self._top_cover_volume_fraction
-
         self._manure_in_digester.ammoniacal_nitrogen = min(
             self._manure_in_digester.ammoniacal_nitrogen * TAN_INCREASE_FACTOR, self._manure_in_digester.nitrogen
         )
 
+        methane_density = 16.04 / (0.0821 * (self._temperature_set_point + 273.15))
+        carbon_dioxide_density = 44.01 / (0.0821 * (self._temperature_set_point + 273.15))
```

Comment:
Should these numbers be saved into some constants?
Like:
- 44.01 -> molar mass of CO2?
- 16.04 -> manure density at STP?
- 0.0821 -> ideal gas constant
```suggestion
        methane_density = 16.04 / (0.0821 * (self._temperature_set_point + GeneralConstants.CELSIUS_TO_KELVIN))
        carbon_dioxide_density = 44.01 / (0.0821 * (self._temperature_set_point + GeneralConstants.CELSIUS_TO_KELVIN))
```